src/components/CourseCard/CourseCard.jsx

```diff
@@ -43,10 +46,22 @@ const CourseCard = ({
 				</div>
 				<div>Duration: {timeConverter(duration)} hours</div>
 				<div>Created: {creationDate} hours</div>
-				<Button>Show course</Button>
+				<Link to={`/courses/${id}`}>
```

Comment:
wrap `a` tag with `button` tag is a bad practice from a11y and semantic points of view